pyproject.toml

```
@@ -1,64 +1,70 @@
 [build-system]
-requires = ["setuptools>=68", "wheel", "setuptools_scm>=6.2,<8", "setuptools-git-versioning<2"]
 build-backend = "setuptools.build_meta"
+requires = ["setuptools>=68", "wheel", "setuptools_scm[toml]>=7.0", "versionista>=1.1.0"]
 
 [project]
-name = "psqlgraph"
-dynamic = [
-  "version",
-  "description",
-  "readme",
-  "requires-python",
-  "license",
-  "authors",
-  "maintainers",
-  "keywords",
-  "classifiers",
-  "urls",
-  "scripts",
-  "gui-scripts",
-  "entry-points",
-  "dependencies",
-  "optional-dependencies"
+authors = [{name = "NCI GDC", email = "support@nci-gdc.datacommons.io"}]
+classifiers = [
+  "License :: OSI Approved :: Apache Software License",
+  "Operating System :: Unix",
+  "Operating System :: POSIX :: Linux",
+  "Topic :: Internet",
+  "Topic :: Software Development :: Libraries :: Python Modules",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: 3.13",
+  "Development Status   :: 5 - Production/Stable"
 ]
+dependencies = ["psycopg2", "rstr", "sqlalchemy>=1.3,<1.4", "xlocal"]
```

I thought sqlalchemy was updated already or is it incoming?
```suggestion
dependencies = ["psycopg2", "rstr", "sqlalchemy==1.4", "xlocal"]
```

---

That is the end goal. This sets the stage for the shift.